### PR TITLE
Add Subham Dutta card info

### PR DESCRIPTION
**Pull request by Sduttt**
Added a card and modified it with my info.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+        <!-- ________ *Subham Dutta* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Subham Dutta</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@Subhamd88404337</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/Sduttt" target="_blank">Sduttt</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/subham-dutta-8670b8178/" target="_blank">Subham</a>
```
Comment (TenTraicion) [inline index.html:126]:
```suggestion
            <a href="https://www.twitter.com/Subhamd88404337" target="_blank">@Subhamd</a>
            <i class="fab fa-github"></i>
            <a href="https://github.com/Sduttt" target="_blank">@Sduttt</a>
            <i class="fab fa-linkedin"></i>
            <a href="https://www.linkedin.com/in/subham-dutta-8670b8178/" target="_blank">@Subham</a>
```

File: index.html [hunk 2]
```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">I'm a web-developer from India. Currently I'm learning Full-stack javascript Development.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="freecodecamp.org" target="_blank" title="Freecodecamp">Freecodecamp</a>
```
Comment (TenTraicion) [inline index.html:133]:
```suggestion
                <a href="https://freecodecamp.org" target="_blank" title="Freecodecamp">Freecodecamp</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Sduttt, Thanks for the contribution. Kindly make the mentioned changes and commit.

**Review by CandidDeer — APPROVED** [review]

**Comment by CandidDeer** [comment]
Thank you for your contribution and making the requested changes. I have merged your PR and your card will be live in a moment.

**Pull request merged**